tests/zeek/replaces.zeek

```diff
@@ -1,5 +1,9 @@
-# @TEST-EXEC: ${ZEEK} -r ${TRACES}/ssh-single-conn.trace  ssh.spicy ./ssh.evt %INPUT | sort >output
+# @TEST-EXEC: mkdir -p modules
+# @TEST-EXEC: spicyz -o modules/ssh.hlto ssh.spicy ./ssh.evt
+# @TEST-EXEC: ZEEK_SPICY_MODULE_PATH=$(pwd)/modules ${ZEEK} -r ${TRACES}/ssh-single-conn.trace %INPUT | sort >output
 # @TEST-EXEC: btest-diff output
+#
+# We use the module search path for loading here as a regression test for #137.
```

Comment:
I was able to confirm that the changes lead to correct behavior by testing in our Docker image (with patched spicy-plugin).

I tried this test without the plugin changes, and at least on my machine (macos, Zeek and Spicy from Homebrew) this test always succeeds.

Reply:
I had tested this test locally without the change, and it failed as expected for me.

Reply:
Yepp, this only fails if one uses a builtin spicy-plugin. Maybe we should add a comment.